loopd/swapclient_server.go

```diff
@@ -78,6 +79,11 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 		}
 	}
 
+	// Check that the label is valid.
+	if err := labels.Validate(in.Label); err != nil {
```

Comment:
nit and pre-existing: most of the logic in Validate could be replaced by `strings.HasPrefix()`.